### ISSUE TITLE
array type

### ISSUE THREAD

**Issue by kondaurovDev** [eclipsesource/play-json-schema-validator#40]
Hi! I'm playing with this validator at http://play-json-schema-validator.herokuapp.com/

When schema is:
```
{
  "type": "array"
}
```

and json instanse:
`[1,2,3]`

I get strange validation error:
```
[
    1,
    2,
    3,
    1,
    2,
    3
]
```

What am i doing wrong? I just want to check that json is an array

**Comment by edgarmueller** [comment]
Hi,
thanks for the report! This is not an error of the validator itself but of the web front-end. Therefore I created an issue for it [here](https://github.com/edgarmueller/schema-validator-web/issues/1).
I'll try to fix this soon. Thanks again!

**Comment by kondaurovDev** [comment]
Aaa, that's good, i was afraid that's a bug. I've started to use this validator since today and it seems like it works as expected :) I was trying to find such a lib but wasn't successful till yesterday. There's a java lib but it has no maintainer and it's not scala anyway. I was really surprised that there are so many implementation even in javascript, but coudn't find java/scala :) Thanks for job!

**Comment by edgarmueller** [comment]
Thanks for the kind words! 
Unfortunately, after a little bit of more debugging today, I also found an error in the lib itself which only applies when merging error information for arrays. Hence, the duplicate items. 
I'll update the web frontend as well the library (0.6.5) until the end of this week.

**Comment by edgarmueller** [comment]
Fixed with 0.6.5.
